policygenerator/policy-sets/community/openshift-plus-setup/machine-sets.yaml

```
@@ -149,7 +149,12 @@ spec:
                   numCPUs: '{{ fromConfigMap "policies" "opp-settings" "vmwareCPU" | toInt }}'
                   numCoresPerSocket: 4
                   snapshot: ""
-                  template: {{ (lookup "config.openshift.io/v1" "Infrastructure" "" "cluster").status.infrastructureName }}-rhcos-generated-region-generated-zone
+                  {{- $tmpsuffix := "region-generated-zone" }}
+                  {{- $ocpversion := semver (fromClusterClaim "version.openshift.io") }}
+                  {{- if gt $ocpversion.Minor 18 }}
```

This can be consolidated using `semverCompare`
```suggestion
                  {{- if semverCompare ">1.18"  (fromClusterClaim "version.openshift.io") }}
```

---

I adopted this.